dataland-frontend/src/components/general/SlideShow.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div
+    ref="slider"
+    role="list"
+    :class="slidesContainerClasses"
+    @pointerdown="dragStartCondition"
+    @touchstart="dragStartCondition"
+  >
+    <slot />
+  </div>
+  <div v-if="slideCount > 1" :class="arrowsContainerClasses">
+    <button @click="move(-1)" aria-label="Previous slide" :class="leftArrowClasses" />
+    <button @click="move(1)" aria-label="Next slide" :class="rightArrowClasses" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onUnmounted, ref, defineEmits, toRefs, withDefaults } from "vue";
```

Comment:
When building the frontend for dev, vite/vue logs this:

![image](https://github.com/d-fine/Dataland/assets/99255296/fc016a51-bfcb-48de-8ba5-1e701d805074)


So it seems defineEmits does not require an import statement.

Reply:
Correct, I removed it